pkg/cmd/cli/install/install.go

```diff
@@ -164,9 +164,6 @@ func NewInstallOptions() *Options {
 		DefaultSnapshotMoveData:  false,
 		DisableInformerCache:     false,
 		ScheduleSkipImmediately:  false,
-		MaintenanceCfg: repository.MaintenanceConfig{
-			KeepLatestMaitenanceJobs: repository.DefaultKeepLatestMaitenanceJobs,
-		},
 	}
```

Comment:
This value is propagated from `velero install` to `velero server`, if `velero server` checked the value not set, it will assigned with  `DefaultKeepLatestMaitenanceJobs`
so it no need to explicitly specify the default parameter values in `velero install` again.